lib/cn/csched_sp3.c

```diff
@@ -412,65 +412,27 @@ sp3_log_progress(struct cn_compaction_work *w, struct cn_merge_stats *ms, bool f
-        SLOG_FIELD("vblk_read1wait_ns", "%ld", ms->ms_vblk_read1_wait.op_time),
-
-        SLOG_FIELD("vblk_read2_ops", "%ld", ms->ms_vblk_read2.op_cnt),
-        SLOG_FIELD("vblk_read2_sz", "%ld", ms->ms_vblk_read2.op_size),
-        SLOG_FIELD("vblk_read2_ns", "%ld", ms->ms_vblk_read2.op_time),
-
-        SLOG_FIELD("vblk_read2wait_ops", "%ld", ms->ms_vblk_read2_wait.op_cnt),
-        SLOG_FIELD("vblk_read2wait_ns", "%ld", ms->ms_vblk_read2_wait.op_time),
-
-        SLOG_FIELD("kblk_read_ops", "%ld", ms->ms_kblk_read.op_cnt),
-        SLOG_FIELD("kblk_read_sz", "%ld", ms->ms_kblk_read.op_size),
-        SLOG_FIELD("kblk_read_ns", "%ld", ms->ms_kblk_read.op_time),
-
-        SLOG_FIELD("kblk_readwait_ops", "%ld", ms->ms_kblk_read_wait.op_cnt),
-        SLOG_FIELD("kblk_readwait_ns", "%ld", ms->ms_kblk_read_wait.op_time),
-
-        SLOG_FIELD("vblk_dbl_reads", "%ld", ms->ms_vblk_wasted_reads),
-
-        SLOG_FIELD("queue_us", "%lu", qt),
-        SLOG_FIELD("prep_us", "%lu", pt),
-        SLOG_FIELD("build_us", "%lu", bt),
-        SLOG_FIELD("commit_us", "%lu", ct),
-        SLOG_END);
+    log_info("cn_comp_stats type=%s job=%u comp=%s rule=%s cnid=%lu nodeid=%lu leaf=%u pct=%3.1f "
+        "vrd_eff=%.3f kblk_alloc_ops=%ld kblk_alloc_sz=%ld kblk_alloc_ns=%ld kblk_write_ops=%ld "
+        "kblk_write_sz=%ld kblk_write_ns=%ld vblk_alloc_ops=%ld vblk_alloc_sz=%ld "
+        "vblk_alloc_ns=%ld vblk_write_ops=%ld vblk_write_sz=%ld vblk_write_ns=%ld "
+        "vblk_read1_ops=%ld vblk_read1_sz=%ld vblk_read1_ns=%ld vblk_read1wait_ops=%ld "
+        "vblk_read1wait_ns=%ld vblk_read2_ops=%ld vblk_read2_sz=%ld vblk_read2_ns=%ld "
+        "vblk_read2wait_ops=%ld vblk_read2wait_ns=%ld kblk_write_ops=%ld kblk_write_sz=%ld "
+        "kblk_write_ns=%ld kblk_readwait_ops=%ld kblk_readwait_ns=%ld vblk_dbl_reads=%ld "
+        "queue_us=%lu prep_us=%lu build_us=%lu commit_us=%lu", msg_type, w->cw_job.sj_id,
+        cn_action2str(w->cw_action), cn_rule2str(w->cw_rule), w->cw_tree->cnid,
+        w->cw_node->tn_nodeid, (uint)cn_node_isleaf(w->cw_node), 100 * progress,
+        vblk_read_efficiency, ms->ms_kblk_alloc.op_cnt, ms->ms_kblk_alloc.op_size,
+        ms->ms_kblk_alloc.op_time, ms->ms_kblk_write.op_cnt, ms->ms_kblk_write.op_size,
+        ms->ms_kblk_write.op_time, ms->ms_vblk_alloc.op_cnt, ms->ms_vblk_alloc.op_size,
+        ms->ms_vblk_alloc.op_time, ms->ms_vblk_write.op_cnt, ms->ms_vblk_write.op_size,
+        ms->ms_vblk_write.op_time, ms->ms_vblk_read1.op_cnt, ms->ms_vblk_read1.op_size,
+        ms->ms_vblk_read1.op_time, ms->ms_vblk_read1_wait.op_cnt, ms->ms_vblk_read1_wait.op_time,
+        ms->ms_vblk_read2.op_cnt, ms->ms_vblk_read2.op_size, ms->ms_vblk_read2.op_time,
+        ms->ms_vblk_read2_wait.op_cnt, ms->ms_vblk_read2_wait.op_time, ms->ms_kblk_read.op_cnt,
+        ms->ms_kblk_read.op_size, ms->ms_kblk_read.op_time, ms->ms_kblk_read_wait.op_cnt,
+        ms->ms_kblk_read_wait.op_time, ms->ms_vblk_wasted_reads, qt, pt, bt, ct);
```

Comment:
Break this up to make it readable, e.g., each line of vars should match up with a separate line of format..